src/scripts/ui/PlayerTradeComponent.tsx

```diff
@@ -149,9 +149,11 @@ export function PlayerTradeComponent({ gameState, xy }: IBuildingComponentProps)
             ]}
             data={trades.filter(
                (trade) =>
-                  resourceFilters.size === 0 ||
-                  resourceFilters.has(trade.buyResource) ||
-                  resourceFilters.has(trade.sellResource),
+                  resources.includes(trade.buyResource) &&
```

Comment:
I don't think it's necessary to limit the trade to what you have unlocked. I will merge and remove this

Reply:
Ah, sorry.  I thought you were in agreement with not letting the player buy resources they had not unlocked.